CONTRIBUTING.md

```diff
@@ -5,17 +5,19 @@ If you are new to contributing to open source, please read the Open Source Guide
 
 ## Table of Contents
 
-* [Code of Conduct](#code-of-conduct)
-* [Ways to Contribute](#ways-to-contribute)
-* [Our Development Process](#our-development-process)
-    * [Issues](#issues)
-    * [Pull Requests](#pull-requests)
-    * [Branching Strategy](#branching-strategy)
-* [Contributing Code](#contributing-code)
-* Internships
-    * [GSoC](#gsoc)
-    * [GitHub Externship](#github-externship)
-* [Community](#community)
```

Comment:
All the changes in the table of contents are unnecessary. Please revert this section.

Reply:
OK, I'll make revert that change.